helpers/common-handler.js

```diff
@@ -281,6 +281,7 @@ function createImprovementTemplate(obj) {
 
         var impTemplatesData = {
             title: obj.title,
+            externalId:Math.floor(new Date() / 1000),
```

Comment:
@rakeshSgr - External ID will be provided from CSV itself, so why set it to a random value ?